src/Interpreters/InterpreterFactory.cpp

```diff
@@ -201,6 +203,10 @@ std::unique_ptr<IInterpreter> InterpreterFactory::get(ASTPtr & query, ContextMut
     {
         return std::make_unique<InterpreterShowIndexesQuery>(query, context);
     }
+    else if (query->as<ASTShowSettingQuery>())
```

Comment:
Needs docs in docs/en/sql-reference/statements/show.md

Reply:
Added.